examples/context/theme-example.js

```diff
@@ -0,0 +1,42 @@
+const defaultTheme = 'light';
+// highlight-next-line
+const ThemeContext = React.createContext(defaultTheme);
+
+class ThemeProvider extends React.Component {
+  state = {theme: 'light'};
```

Comment:
Maybe show a different value than the default?
Maybe don't show a class component with `state` at all? Because it isn't obvious why it's necessary in this case.

Reply:
Actually copied this from the blog post: https://github.com/reactjs/reactjs.org/pull/587/files#diff-551ec185a5d5f7818d2b02fbbf901c28

I'll simplify this one since the dynamic example covers stateful providers.

Reply:
😆 That's fair!